website/content/en/docs/kubebuilder/references/logging.md

```diff
@@ -0,0 +1,273 @@
+---
+title: Kubebuilder Logging
```

Comment:
Why the name of it should be Kuebebuilder Logging? 

Also, should we really doc all as Kubebuider? Is not it strange image that you try to use a tool and then when we check its doc all information is about another similar tool? The base layout with the integration is for now equals kubebuilder but will not it be customized and things will not be added on top? Is not it now an SDK content? 

c/c @estroz @hasbro17 @joelanford

Reply:
Yeah agreed, there's no reason for our docs to phrase things as `Kubebuilder XYZ`.
Even in the quickstart guide the only time I mention Kubebuilder is when I'm referring to upstream docs.

And the logger comes from the controller-runtime so it's not even kubebuilder specific.